Assets/Python/OOSLogger.py

```
@@ -1,190 +1,453 @@
+    fsenc = sys.getfilesystemencoding() or 'mbcs'
+    user_dir = SP.userDir
+    if not isinstance(user_dir, unicode):
+        try:
+            user_dir = user_dir.decode(fsenc, 'replace')
+        except Exception:
+            user_dir = user_dir.decode('latin-1', 'replace')
+    log_dir = os.path.join(user_dir, u"Logs")
+    if not os.path.isdir(log_dir):
+        try:
+            os.makedirs(log_dir)
+        except OSError, e:
```

Ruff (0.12.2)

SyntaxError: Multiple exception types must be parenthesized on Python 3.13 (syntax was added in Python 3.14)

---

Undefined name `e`

(F821)